packages/application-components/src/components/tab-header/tab.styles.ts

```diff
@@ -42,7 +42,7 @@ export const getLinkStyles = (
   isActive &&
     css`
       ${getBottomBorderStyles(customProperties.colorPrimary)}
-      > * > * {
+      > * {
```

Comment:
nitpick: perhaps we can take this opportunity to be more specific with this rule.
Since we now we are using an `h4` element within the link (and that's the element we want to address the color rule to), we could update it to be something like this:
```
& h4 {
```

Reply:
thanks 👍 done in 8a752503b89e5e7125b3540d8d9174ed62840486